### PR TITLE
kyber: update to round 3 spec

### LINKED ISSUE #161: Update Kyber to Round 3

### PR DESCRIPTION
**Pull request by bwesterb**
Closes #161

### CODE REVIEW

File: pke/kyber/kyber1024/internal/cpapke.go [hunk 1]
```diff
@@ -125,12 +125,12 @@ func (pk *PublicKey) EncryptTo(ct, seed, pt []byte) {
...
 
 	// Sample r, e₁ and e₂ from B_η
-	rh.DeriveNoise(seed, 0)
+	rh.DeriveNoise(seed, 0, Eta1)
 	rh.NTT()
 	rh.BarrettReduce()
 
-	e1.DeriveNoise(seed, K)
-	e2.DeriveNoise(seed, 2*K)
+	e1.DeriveNoise(seed, K, 2)
```
Comment (armfazh) [inline pke/kyber/kyber1024/internal/cpapke.go:132]: Eta1?

**Review by armfazh — APPROVED** [review]

File: pke/kyber/kyber1024/internal/cpapke.go (hunk 1 above)
Comment (bwesterb) [reply pke/kyber/kyber1024/internal/cpapke.go:132]: No, this is Eta2 which equals 2 for all parameters.

Comment (armfazh) [reply pke/kyber/kyber1024/internal/cpapke.go:132]: please add a comment in the code, so we can remember this

Comment (bwesterb) [reply pke/kyber/kyber1024/internal/cpapke.go]: I've added a variable.

**Pull request merged**